tests/rados/pool_tests.py

```diff
@@ -77,7 +77,7 @@ def run(ceph_cluster, **kw):
         time.sleep(25)
 
         # Waiting for up to 2.5 hours for the recovery to complete and PG's to enter active + Clean state
-        end_time = datetime.datetime.now() + datetime.timedelta(seconds=9000)
+        end_time = datetime.datetime.now() + datetime.timedelta(seconds=15000)
```

Comment:
Why do we need such a huge timeout? If recovery is delayed, we can switch the mclock profile to `high_recovery_ops`

Reply:
Since this was planned on BM nodes, the scale of cluster needs lots of time to get into active + clean state.

The recovery settings are already modified in the test case to have lot's of recovery threads.